packages/sx05re/emuelec/bin/emuelecRunEmu.sh

```diff
@@ -8,6 +8,8 @@
 
 # This whole file has become very hacky, I am sure there is a better way to do all of this, but for now, this works.
 
+blank_buffer
```

Comment:
@Kelvfimer this is old code from emuelecRunEmu.sh you need to update your branch to the latest dev version. Cheers.